Mathlib/RingTheory/TwoSidedIdeal/Operations.lean

```diff
@@ -248,6 +245,32 @@ lemma mem_asIdealOpposite {I : TwoSidedIdeal R} {x : Rᵐᵒᵖ} :
   simpa [asIdealOpposite, asIdeal, TwoSidedIdeal.mem_iff, RingCon.op_iff] using
     ⟨I.ringCon.symm, I.ringCon.symm⟩
 
+/-- Bundle an `Ideal` that is already two-sided as a `TwoSidedIdeal`. -/
+def ofIdeal (I : Ideal R) (mul_mem_right : ∀ {x y}, x ∈ I → x * y ∈ I) : TwoSidedIdeal R :=
+  TwoSidedIdeal.mk' I I.zero_mem I.add_mem I.neg_mem (I.smul_mem _) mul_mem_right
```

Comment:
I think that this name is not meaningfully different from `fromIdeal` (i.e., without checking the types, I won't know which one is which). I think `_root_.Ideal.toTwoSided` or `_root_.Ideal.toTwoSidedIdeal` would be better. I realize that this name might also suffer slightly from the problem I just mentioned *but*, unlike `fromIdeal`, which can't take advantage of dot notation because it's an order hom, this declaration can do so. This could be the distinguishing feature.

Obviously lemmas below would need to be renamed too, and I'm open to other suggestions here.

Reply:
I agree that having dot notation work is nice. For that reason, should `ofIdeal` also live in the `Ideal` namespace, and perhaps be renamed to `Ideal.twoSidedSpan` or something like that (in another PR!)?

Reply:
Maybe renaming it to `TwoSidedIdeal.spanIdeal` would make sense though.